.gitignore

```diff
@@ -10,3 +10,4 @@ perf.old
 mutants.out
 mutants.out.old
 .cargo/config.toml
+test/
```

Comment:
If you personally have testdata in `./test` in the source tree you can add it in `.git/info/exclude` (https://git-scm.com/docs/gitignore) but this doesn't seem generally needed?

Reply:
Good point, didn't know the existence of `.git/info/exclude`.

Reply:
@sourcefrog should have been reverted.